napari/_qt/qt_viewer.py

```diff
@@ -462,7 +462,7 @@ def _weakref_if_possible(self, obj):
             From https://docs.python.org/3/library/weakref.html: "Objects which
             support weak references include class instances, functions written
             in Python (but not in C), instance methods, sets, frozensets, some
-            file objects, generators, type objects, sockets, arrays, deques,
+            file objects, generators, type objects, sockets, arrays, dequeues,
```

Comment:
```suggestion
            file objects, generators, type objects, sockets, arrays, deques,
```
Revert, https://docs.python.org/3/library/collections.html#collections.deque

Reply:
chatgpt says (llama agrees)

> The plural of "dequeue" would be "dequeues." In English, we typically form the plural of most nouns by adding an "s" or "es" to the end of the word. Since "dequeue" is a term often used in computer science to refer to the operation of removing an element from a queue, the same pluralization rule applies, resulting in "dequeues."

Reply:
Sorry but ChatGPT does not trump the official python docs!
We are not removing from a queu; the singular is not `dequeue` but `deque`, derived from `double-ended queue`. 
I linked the python docs which use `deques` as the plural.

Reply:
makes total sense, thank you @psobolewskiPhD for explanation and patience ;)
- https://github.com/codespell-project/codespell/pull/3649